packages/types/action-client.ts

```diff
@@ -9,6 +9,18 @@ export const ZResource = z.enum([
   "response",
   "survey",
   "person",
+  "tag",
+  "responseNote",
+  "membership",
+  "attributeClass",
+  "segment",
+  "actionClass",
+  "integration",
+  "webhook",
+  "apiKey",
+  "subscription",
+  "invite",
```

Comment:
I'm not sure if `invite` and `membership` needs to be different. When people have `membership, create` rights that also means that they need to be able to create an invite.

Reply:
Yeah, the meaning is identical mostly, but since they possess different schemas and may have more granular control over them (for updates, etc.), I added permissions for other entities.